### ISSUE TITLE
scrappie: error while loading shared libraries: libopenblas.so.0: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by egeogretmen** [nanoporetech/scrappie#19]
Hello,

I've installed scrappie using the bioconda channel, but I'm getting an error regarding libopenblas whichever command I run
`scrappie: error while loading shared libraries: libopenblas.so.0: cannot open shared object file: No such file or directory`
Do you know what it is about and how to solve it?

Best,
Ege

**Comment by tmassingham-ont** [comment]
Hello.  I'm afraid the Bioconda channel was provided by a third party and is not associated with Oxford Nanopore Technologies so we cannot provide support.

The error appears to be something to do with Bioconda recipe failing to install the required `openblas` library; you may be able to install this using your system package manager.

**Comment by wdecoster** [comment]
Perhaps not associated with ONT so you are not required to support this - but it would be hugely beneficial to simplify the installation.

The bioconda [recipe](https://github.com/bioconda/bioconda-recipes/pull/8456/files) for scrappie specifies `blas` as a dependency. Should that be `openblas`?

**Comment by tmassingham-ont** [comment]
`openblas` provides a `blas` implementation and there appears to be a conda channel to install openblas.  I'm not sure whether the installation would require `openblas` instead of `blas` or as well as.

**Comment by wdecoster** [comment]
I reproduced the issue of lacking openblas when installing from bioconda.

The scrappie recipe passed the automatic testing when it was added to bioconda a couple of months ago. I just reran that test, and this time it failed.

I replaced blas with openblas as a dependency, and the recipe passed testing (on linux). Therefore I have now changed the bioconda recipe and will merge it after it passes the macos test. It should be available for installation then later today I guess.

See also https://github.com/bioconda/bioconda-recipes/pull/9744
